src/pages/deployment/builds.md

```diff
@@ -92,8 +98,7 @@ process1: command
 process2: command
```

Comment:
Might want to remove this bit too since we don't support multiple processes

Reply:
Lemme fix that